packages/checkout/widgets-lib/src/components/NotEnoughGas/NotEnoughGas.tsx

```diff
@@ -58,86 +58,63 @@ NotEnoughGasProps) {
+            top: 'base.spacing.x5',
+            left: 'base.spacing.x5',
+            backdropFilter: 'blur(30px)',
+          }}
+          onClick={onCloseDrawer}
+        />
+        <Heading
+          size="small"
+          weight="bold"
+          sx={headingTextStyles}
+          testId="not-enough-gas-heading"
+        >
+          {heading}
+        </Heading>
+        <Body sx={bodyTextStyles}>
+          {body}
+        </Body>
+        <Box sx={actionButtonContainerStyles}>
+          {tokenSymbol === ETH_TOKEN_SYMBOL
+            ? (
+              <Button
+                testId="not-enough-gas-copy-address-button"
+                sx={actionButtonStyles}
+                variant="primary"
+                onClick={handleCopy}
+              >
+                {t('drawers.notEnoughGas.buttons.copyAddress')}
+                <Button.Icon icon={isCopied ? 'Tick' : 'CopyText'} />
+              </Button>
+            )
+            : (
               <Button
-                testId="not-enough-gas-adjust-amount-button"
+                testId="not-enough-gas-add-imx-button"
                 sx={actionButtonStyles}
-                variant="tertiary"
-                onClick={onCloseDrawer}
+                variant="primary"
+                onClick={onAddCoinsClick}
               >
-                {t('drawers.notEnoughGas.buttons.adjustAmount')}
+                {t('drawers.notEnoughGas.buttons.addMoreImx')}
               </Button>
             )}
-            {
-              tokenSymbol === ETH_TOKEN_SYMBOL
-                ? (
-                  <Button
-                    testId="not-enough-gas-copy-address-button"
-                    sx={actionButtonStyles}
-                    variant="tertiary"
-                    onClick={handleCopy}
-                  >
-                    {t('drawers.notEnoughGas.buttons.copyAddress')}
-                    <Button.Icon icon={isCopied ? 'Tick' : 'CopyText'} />
-                  </Button>
-                )
-                : (
-                  <Button
-                    testId="not-enough-gas-add-imx-button"
-                    sx={actionButtonStyles}
-                    variant="tertiary"
-                    onClick={onAddCoinsClick}
-                  >
-                    {t('drawers.notEnoughGas.buttons.addMoreImx')}
-                  </Button>
-                )
-            }
-            <Button
-              sx={actionButtonStyles}
-              variant="tertiary"
-              onClick={onCloseDrawer}
-              testId="not-enough-gas-cancel-button"
-            >
-              {t('drawers.notEnoughGas.buttons.cancel')}
-            </Button>
-          </Box>
-          <Box sx={logoContainerStyles}>
-            <Logo
-              testId="footer-logo-image"
-              logo="ImmutableHorizontalLockup"
-              sx={{ width: 'base.spacing.x25' }}
-            />
-          </Box>
         </Box>
+        <FooterLogo />
```

Comment:
Not sure if this is part of the design. For the drawers that I created, the figma designs didn't have the footer logo 🤔

Reply:
have removed it now